adapters/smaato/smaato.go

```diff
@@ -34,6 +34,11 @@ type adapter struct {
 	endpoint string
 }
 
+// impExt defines Imp.Ext object for Smaato
+type impExtData struct {
+	Skadn json.RawMessage `json:"skadn,omitempty"`
+}
+
```

Comment:
this can be moved to `openrtb_ext` package in [imp_smaato.go](https://github.com/prebid/prebid-server/blob/de6ed827885e8ab02efd8d94b7b16157bd4d9538/openrtb_ext/imp_smaato.go#L1) file

Reply:
@onkarvhanumante I'm not really sure if I can do that. `Skadn` is not part of `openrtb_ext.ExtImpSmaato (ext.imp.bidder)`. But rather `openrtb_ext.ExtImpSmaato` could be a `bidder` field inside this struct `(ext.imp)`, alongside `Skadn`.
What is your suggestion on this?

Reply:
Moved the struct. However, for the existing one [docs](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-parameters-code) mention we should keep it.